sunpy/time/julian.py

```diff
@@ -12,13 +12,10 @@ def julian_day(t='now'):
     """
     return Time(parse_time(t)).jd
 
-# The number of days between Jan 1 1900 and the Julian reference date of
-# 12:00 noon Jan 1, 4713 BC
-JULIAN_DAY_ON_NOON01JAN1900 = 2415021.0
-
 
 def julian_centuries(t='now'):
-    """Returns the number of Julian centuries since 1900 January 0.5."""
-    DAYS_IN_YEAR = 36525.0
+    """Returns the number of Julian centuries since J1900.0 (noon on 1900 January 0)."""
+    DAYS_IN_JULIAN_CENTURY = 36525.0
```

Comment:
Is it TT or TD? - In my PR I am using what's said in the book - which it changes from equation to equation. So I would suggest to be careful with this particular bit.